status/raw/status-app-protocols.md

```diff
@@ -0,0 +1,372 @@
+
+Full clients SHOULD prefer relay protocol to subscribe to pubsub topics matching the scopes:
+1. Global control
+2. Global content
+3. Community control, for each community of which the app user is a member
+4. Community content, for each community of which the app user is a member
+
+Light clients SHOULD use filter protocol to subscribe only to the content topics relevant to the user.
+
+**Self-addressed messages**
+
+Status clients (full or light) MUST NOT subscribe to topics for messages with self-addressed scopes.
+See [Self-addressed messages](#self-addressed-messages).
+
+**Large messages**
+
+Status clients (full or light) SHOULD NOT subscribe to topics set aside for large messages.
+See [Large messages](#large-messages).
+
+### Publishing
+
+The application MUST publish user and app generated messages via the Waku transport layer.
+The specific Waku protocol used for publishing depends on the resource-availability of the client:
+1. Lightpush protocol, as specified in [19/WAKU2-LIGHTPUSH](../../waku/standards/core/19/lightpush.md) allows publishing to a pubsub topic via an intermediate "full node" and is more appropriate for resource-restricted publishing.
+2. Relay protocol, as specified in [11/WAKU2-RELAY](../../waku/standards/core/11/relay.md), allows publishing directly into the relay routing network and is therefore more resource-intensive.
+
+Full clients SHOULD use relay protocol to publish to pubsub topics matching the scopes:
+1. Global control
+2. Global content
+3. Community control, for each community of which the app user is a member
+4. Community content, for each community of which the app user is a member
+
+Light clients SHOULD use lightpush protocol to publish control and content messages.
+
+**Self-addressed messages**
+
+Status clients (full or light) MUST use lightpush protocol to publish self-addressed messages.
+See [Self-addressed messages](#self-addressed-messages).
+
+**Large messages**
+
+Status clients (full or light) SHOULD use lightpush protocols to publish to pubsub topics set aside for large messages.
+See [Large messages](#large-messages).
+
+### Retrieving historical messages
+
+Status clients SHOULD use the store query protocol, as specified in [WAKU2-STORE](https://github.com/waku-org/specs/blob/8fea97c36c7bbdb8ddc284fa32aee8d00a2b4467/standards/core/store.md), to retrieve historical messages relevant to the client from store service nodes in the network.
+
+Status clients SHOULD use [content filtered queries](https://github.com/waku-org/specs/blob/8fea97c36c7bbdb8ddc284fa32aee8d00a2b4467/standards/core/store.md#content-filtered-queries) with `include_data` set to `true`,
+to retrieve the full contents of historical messages that the client may have missed during offline periods,
+or to populate the local message database when the client starts up for the first time.
+
+**Store queries for reliability**
```

Comment:
I think we should clean up reliability specs (my bad, I committed to do it and never got around to it) so we can refer to them here instead of restating logic.

Reply:
That will be great.